README.md

```diff
@@ -27,67 +27,140 @@ collection <- "<collection_name>"
 mongodb_uri <- "<mongodb://uri>"
 
 # Création d'une tâche d'apprentissage
+# C'est un objet de type "S3", donc une liste dans laquelle sont stockées les 
+# informations de la tâche d'apprentissage
 task <- sf_task(
   mongodb_uri,
   database,
   collection
 )
 
-last_batch <- "2005_2" # Nom du dernier batch importé
-sample_size <- 10000 # Échantillonnage pour l'entraînement de 10.000 couples (siret x période). 
-# (en réalité 800.000 en production)
+last_batch <- "2009_5" # Nom du dernier batch importé
 
-# Chargement des données historiques
+# Échantillonnage aléatoire pour l'entraînement de 10.000 
+# couples (siret x période).  (en réalité 800.000 en production)
+sample_size <- 10000 
+
+# Chargement des données historiques (par défaut, 2015 et 2016)
 task <- load_hist_data(
   task = task,
   batch = last_batch,
   subsample = sample_size
 )
 
-# On entraîne sur toute la base.
-task <- split_data(task, resampling_strategy = "none") 
 
 # Utilitaire pour lister les champs
 fields <- get_fields(training = TRUE) 
 
-# Pipeline de préparation des données. task <- prepare(
+# Si l'on entraîne sur toute la base, pas besoin de créer d'échantillons de 
+# test. Se référer au script de test pour voir comment faire. 
+
+# prepare spécifie une suite d'opérations sur les données, sous la forme d'un
+# `mlr3pipelines::Graph` ou assimilable. 
+# Les opérations ne seront effectuées qu'au moment de l'entraînement.
+task <- prepare(
   task,
-  training_fields = training_fields
+  training_fields = fields
 )
 
+# Si l'on veut tout de même consulter l'effet de la préparation sans entraîner 
+# de modèle, on peut utiliser:
+prepared_data <- get_prepared_data(task)
+
 # Entraînement avec le modèle par défaut (xgboost)
 task <- train(task)
 # Ou alors avec un autre modèle (issu de mlr3 ou mlr3learners)
 task <- train(task, learner = mlr3::lrn("classif.rpart"))
 
 # Chargement de nouvelles données
-last_period <- as.Date("2020-01-01")  # Dernière période importée task <- 
-load_new_data(
+last_period <- as.Date("2020-01-01")  # Dernière période importée
+task <- load_new_data(
```

Comment:
```
ask <- load_new_data(
   task = task,
   periods = last_period,
   batch = last_batch
 )
```
INFO  [17:28:09.286] Loading data from last batch 
INFO  [17:28:09.343] Connexion a la collection mongodb Features ... 
Error in import_data(database = database, collection = collection, mongodb_uri = mongodb_uri,  : 
  promise already under evaluation: recursive default argument reference or earlier problems?

Reply:
Tous ces commentaires arrivent quand je fais tourner le code depuis le serveur

Reply:
Normalement fixée par la PR #43